### PR TITLE
Added rough README for rshiny quickstarter....

### PR DESCRIPTION
**Pull request by sklingel**
.... since I have never used it and have almost zero experience with shiny. Could someone who actually uses it provide additional information?

### CODE REVIEW

File: boilerplates/rshiny/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+## Purpose of this quickstarter
+Provisions a Rshiny application within openshift using crowd authentification.
+
+## What files / architecture is generated?
++-- Jenkinsfile  
++-- Dockerfile    
++-- app.R
+
+## Frameworks used
+R/Rshiny
```
Comment (clemensutschig) [inline boilerplates/rshiny/README.md]: please add links underneath that provide a tutorial or alike :)

File: boilerplates/rshiny/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Usage - how do you start after you provisioned this quickstarter
+The quickstarter sets up two pods in openshift. The rshiny application is routed through a crowd authentication proxy.
```
Comment (clemensutschig) [inline boilerplates/rshiny/README.md]: please link to the image on github in ods-core/Shared images

Comment (sklingel) [reply boilerplates/rshiny/README.md]: Done in newest commit

File: boilerplates/rshiny/README.md (hunk 1 above)
Comment (sklingel) [reply boilerplates/rshiny/README.md]: Done in newest commit

**Pull request merged**